diktat-gradle-plugin/src/functionalTest/kotlin/org/cqfn/diktat/plugin/gradle/DiktatGradlePluginFunctionalTest.kt

```diff
@@ -0,0 +1,101 @@
+package org.cqfn.diktat.plugin.gradle
+
+import org.cqfn.diktat.plugin.gradle.DiktatGradlePlugin.Companion.DIKTAT_CHECK_TASK
+import org.gradle.internal.impldep.org.junit.rules.TemporaryFolder
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.io.File
+
+class DiktatGradlePluginFunctionalTest {
+    private val testProjectDir = TemporaryFolder()
+    private lateinit var buildFile: File
+
+    @BeforeEach
+    fun setUp() {
+        testProjectDir.create()
+        File("../examples/gradle-kotlin-dsl").copyRecursively(testProjectDir.root)
+        File(testProjectDir.root, "build.gradle.kts").delete()
+        buildFile = testProjectDir.newFile("build.gradle.kts").apply {
+            writeText(
+                """
```

Comment:
Why diKTat is not complaining on this indent?

Reply:
`(` is a token that increases indent by one `INDENT_SIZE`, so it's valid.